samples/keys/hw_unique_key/sample.yaml

```diff
@@ -2,14 +2,15 @@ sample:
   description: Generate and write random keys to the KMU.
   name: Random HW Unique Key
 common:
-    tags: hw_unique_key
-    platform_allow: nrf5340dk_nrf5340_cpuapp nrf5340dk_nrf5340_cpuapp_ns nrf9160dk_nrf9160 nrf9160dk_nrf9160_ns nrf52840dk_nrf52840
+    tags: huk
```

Comment:
The `hw_unique_key` tag is used in `tests/lib/hw_unique_key` and `tests/lib/hw_unique_key_tfm` as well. Please update those as well.

Reply:
Thanks. Updated in testcase.yaml files as well now.